flake.nix

```diff
@@ -8,7 +8,7 @@
     flake-utils.lib.eachDefaultSystem (
       system:
       let
-        pkgs = nixpkgs.legacyPackages.${system};
+        pkgs = import nixpkgs { inherit system; };
```

Comment:
I would prefer the `nixpkgs.legacyPackages.${system}` style. If anyone ever uses our flake in their own it won't cause an unnecessary evaluation.